syntax/problem.php

```diff
@@ -107,14 +111,21 @@ public function render($mode, \Doku_Renderer $renderer, $data) {
         return false;
     }
 
+    /**
+     * Renders content of the task
+     * @param Doku_Renderer $renderer
+     * @param \PluginFKSTaskRepo\Task $data Task data
+     * @param bool $full If the header should contain additional information
+     */
     private function renderContent(Doku_Renderer &$renderer, \PluginFKSTaskRepo\Task $data, $full = false) {
         $renderer->doc .= '<div class="mb-3" data-label="' . $data->getLabel() . '">';
         $this->renderHeader($renderer, $data, $full);
-        $this->renderFigures($renderer, $data);
+        $this->renderImageFigures($renderer, $data);
         $this->renderTask($renderer, $data);
-        // $this->renderSolutions();
+        $this->renderFileAttachments($renderer, $data);
+        $hasSolution = $this->renderSolutions($renderer, $data);
         $this->renderTags($renderer, $data);
-
+        $hasSolution && $this->renderOrigin($renderer, $data);
```

Comment:
radšej použiť if -- funguje to ale využíva sa to hlavne pri last-value jazykoch, (PHP nieje last-value)